terasoluna-gfw-parent/pom.xml

```diff
@@ -470,6 +470,24 @@
       </dependency>
       <!-- == End Apache Tiles == -->
 
+      <!-- == Begin Logback and SLF4J == -->
```

Comment:
- [ ] Please put "tentative support until Spring Boot supports it" in the comments.